### ISSUE TITLE
Internal Review notes

### ISSUE THREAD

**Issue by snblickhan** [zooniverse/scribes-of-the-cairo-geniza#72]
- [x] Updates from #66 still need to be completed 

- [x] Collect & Talk buttons in the menu bar are redirecting to the Home page. Do these need to be connected to PFE? 

HOME PAGE

- [x] Add Bodleian & Haifa e-Lijah lab logos to institutional partners section

- [x] Can we change the terminology on the Front page from "FULL TRANSCRIPTION EXERCISES" to "TRANSCRIBE TEXT"? ![example](http://i65.tinypic.com/4rx5ax.png)

TUTORIAL

- [x] Tutorial -- paragraph breaks aren't showing up, though they are in PFE. 

- [x] Scrolling doesn't work / text is cut off.

- [x] Add "Let's go" button to last slide of tutorial

- [x] Media for Easy Arabic tutorial

- [x] Re-word "Transcribe" slide to mention opening toolbar.

TRANSCRIPTION INTERFACE

- [x] No prompt to continue saved work on sign-in (issue #78)

- [x] "What's This?" modal for text modifiers (#76) 

- [x] Library catalogue links not working/opening in the same window (should be new window).

- [x] Attribution section needs to resize so text doesn't get cut off ![example](http://i65.tinypic.com/1zxvzop.png)

- [x] The "Script References" tab in the Crib Sheet needs to be deactivated in the Arabic workflows.

- [x] Cribsheet thumbnails are sizing oddly -- getting zoomed in a lot (I think sizing to height instead of width), which might not be very useful for finding visual references quickly.

- [x] Pop-up conflict

- [x] Save text appearing below Subject Info box ![example](http://i65.tinypic.com/x6fx8z.png)

KEYBOARD & TRANSCRIPTION BOX

- [x] Transcription box is entirely covering the fragment -- can we set this up so it doesn't do this? Otherwise we'll need to move the box for every single transcription, which we've had negative feedback about on ASM. [possibly wait until after Beta to see what feedback is like?]

- [x] The Lam+Alif key on the Arabic keyboard gives a Lam on the transcription, instead of giving a Lam+Alif   ل   +   ا   (but in reverse order -- J + I)
- If it's in the intial position of a word, it should look like   لا
- It it's in an ending position of a word, it should be  قلا  <-- This is 2 characters, I'm only referring to the character that looks similar to a backward K (I can't generate it on the keyboard without additional text so that it's the end character in a word)

- [x] Show Previous Marks button doesn't do anything 

- [x] Remove Hebrew keyboard option from Arabic transcription workflow 

- [x] Arabic workflow only -- text on transcription box reads "The Hebrew language reads from right to left...&c" -- needs to be changed to "The **Arabic** language reads...&c"

**Comment by VVH** [comment]
**Sign in**
Can't sign in on Chrome. this is a problem I've had on various projects including ShaxWorld, but it's no longer a problem for me there. This might be a legacy of Panoptes CFE

**About Section**
- Sentence on About page 'the ligatures (the way letters connect to each other in Arabic script) makes it" should be 'make it'
- Maybe move 'About the Geniza' section up to the top as more compelling than the current first section.
- 'Phase II.The Goal' on About page just needs a space between II. and The

**Tutorial**
- Various steps of the tutorial say 'Needs Media'
-In penultimate step of tutorial there's an explanation about retirement after 3 people have transcribed. Is this the case or is it that N people transcribe and once there is agreement between 3 the line retires? If the former, that's more like DCW, less like Shakespeare's World and AnnoTate, which might be worth writing a blog post about down the line.

**Transcription Interface**
- If I am transcribing and click on the Field Guide my transcription box is closed automatically. When I click on the dots to reopen the box, my transcription is gone. Should autosave or not close, as in ShaxWorld? 
- Rotate button didn't work. 

Minor formatting weirdness in Finished Box
![cg_finishedbox_2018-05-03 at 16 42 45](https://user-images.githubusercontent.com/6482891/39587992-ba6e126c-4ef2-11e8-8a97-b659d3c7e51a.png)

**Comment by snblickhan** [comment]
Thanks for the feedback, @VVH -- @beckyrother is adding tutorial media today, and several of your comments are already open in #66 (so they're on the radar!).

I'll push back a bit on the tutorial comment: it doesn't say retirement is after 3 people have transcribed, but notes that "Subjects are retired from the project once three people have marked them as complete." Might be that we need to explain what "complete" means (i.e. everything is greyed-out). I followed ShaxWorld on this a bit, as that project uses "complete" without going into more detail about what that means -- did you get much pushback on Talk about that?

**Comment by VVH** [comment]
Ok, yes, I see I didn't represent the content accurately, but I think the point remains. Once three people say something is complete, then aggregation attempts to compare the submissions, and either greys out the line or re-releases it. Re: Shakespeare's World, that tutorial step is essentially out of date (and perhaps we can change the wording of the tutorial--thanks for the prompt on that). You may choose to change retirement rules later, so rather than specifying a number in the tutorial, you could be a bit vague there. It's a minor point, but I raise it on the back of experience with ShaxWorld for which we've changed retirement several times.

**Comment by snblickhan** [comment]
Ahhh okay, I see. That's a really good point, being less specific definitely gives us more room to amend things if we need/want to!

**Comment by beckyrother** [comment]
Couple things here:
- [ ] The Field Guide should open _in addition_ to the transcribe box, not _instead of_.
- [ ] When the transcription box is open, the Pan tool will not "let go" of the image:
![let-go-sm](https://user-images.githubusercontent.com/1392130/39591073-3f54005c-4ec8-11e8-899d-c9ee00542ece.gif)

**Comment by trouille** [comment]
The drop-down selection menu for which workflow to choose seems overly complicated. Could it be simplified to fewer words or other way to make more clear/streamlined? 

<img width="265" alt="screen shot 2018-05-04 at 11 30 03 am" src="https://user-images.githubusercontent.com/4975655/39639343-9b0e5002-4f8e-11e8-8b5a-1b26de16670f.png">

**Comment by trouille** [comment]
Slide 2 of tutorial in 'Easy transcription-hebrew' says click on 'Add a transcription', but at least for me, when I first arrived on the page, the text next to the icons on the left-hand side was collapsed (i.e., only saw the icons). You'll want to update the tutorial so that it doesn't just refer to 'add a transcription' text. 

Also, do you want spacing between paragraphs in your tutorial? And the bottom line in the screenshot maybe butts up a little too closely to the slider-dots? 

And scrolling within the tutorial doesn't seem to be enabled? In slide 3, I can't scroll down. 

Note: I'm on macbook pro high sierra on chrome v66.

<img width="447" alt="screen shot 2018-05-04 at 11 33 59 am" src="https://user-images.githubusercontent.com/4975655/39639901-23bb1890-4f8f-11e8-8b46-fea9474de56b.png">

**Comment by trouille** [comment]
Slide 7 says 'needs media'

<img width="453" alt="screen shot 2018-05-04 at 11 36 35 am" src="https://user-images.githubusercontent.com/4975655/39640072-78c60476-4f8f-11e8-9fe0-2bfeb1372092.png">

And last slide of tutorial doesn't have 'Done', just 'Next', and doesn't quit out without me having to click on the 'X' in the upper right (which is fine, but should be like our other tutorials). 

<img width="466" alt="screen shot 2018-05-04 at 11 38 03 am" src="https://user-images.githubusercontent.com/4975655/39640142-afb81cd0-4f8f-11e8-93f4-d195bc5291ee.png">

**Comment by trouille** [comment]
This may be on purpose, but should the crib sheet thumbnails be so zoomed in? 

<img width="775" alt="screen shot 2018-05-04 at 11 40 32 am" src="https://user-images.githubusercontent.com/4975655/39640261-0541b65c-4f90-11e8-8cd0-8a829761e770.png">

Which shows the thumbnail for:
<img width="455" alt="screen shot 2018-05-04 at 11 41 05 am" src="https://user-images.githubusercontent.com/4975655/39640276-11b0874c-4f90-11e8-9663-721f2330f747.png">

**Comment by trouille** [comment]
Since people can drop dots left to right, will your aggregation code be able to handle this? Could you put in a restriction to not allow left-to-right? I know you'd need to think about angled text and vertical text, but wondered if there were some at least guardrails you could set.

**Comment by trouille** [comment]
'Simple transcription arabic' workflow has lots of 'needs media' in the tutorials.

**Comment by trouille** [comment]
I tried the 'save progress' functionality. I did a couple lines of transcriptions on the easy hebrew workflow. I then saved progress. I then quit out of that browser. I then restarted the browser and signed back into cairo. And I didn't get the prompt to continue my previous effort (as the tutorial suggests).

**Comment by snblickhan** [comment]
Doesn't work for me, either. Have opened a new issue: #78

**Comment by mcbouslog** [comment]
didn't notice anything major not already mentioned, but after a quick try just a few notes:
- on home page, it's unclear where stats bars end, looked a little strange, could just be me
- from classify, had transcribe box open, clicked Scribes of the Cairo Geniza in the upper right nav to go to home and box stayed open, wouldn't close
- when transcribing the simple Hebrew, I presume ok to ignore vowels as there aren't keys for them? prob in tutorial/field guide but I missed

otherwise really cool! Field Guide was super helpful to have open when looking for keywords.
